nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/BulkAdminService.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * (C) Copyright 2018 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *       Kevin Leturc <kleturc@nuxeo.com>
+ */
+package org.nuxeo.ecm.core.bulk;
+
+import java.util.List;
+
+import org.nuxeo.runtime.kv.KeyValueStore;
+
+/**
+ * The Bulk admin service, it's an internal service in order to access configuration from Bulk Operation Framework.
+ *
+ * @since 10.2
+ */
+// This service is needed by stream processors to create appropriate streams
+public interface BulkAdminService {
+
+    /**
+     * @return the {@link KeyValueStore} name used by service.
```

Comment:
Please add a full sentence to all Javadocs. "Gets the key/value store used by the service." here.
No period at the end of `@return` and `@param` lines.